core/dbt/semver.py

```diff
@@ -95,7 +95,7 @@ def from_version_string(cls, version_string):
 
         if not match:
             raise dbt.exceptions.SemverException(
-                'Could not parse version "{}"'.format(version_string)
+                '"{}" is not a valid semantic version.'.format(version_string)
```

Comment:
```suggestion
                f'"{version_string}" is not a valid semantic version.'
```

Reply:
Added this change in the most recent commit. Thanks!